Makefile

```diff
@@ -1,4 +1,5 @@
 BINDIR      := $(CURDIR)/bin
+HELM_INSTALL_DIR ?= /usr/local/bin
```

Comment:
Looking at the [kernel documentation on kbuild](https://www.kernel.org/doc/html/v5.4/kbuild/makefiles.html), `INSTALL_PATH` is the common variable for referencing where binaries should be installed. Can you please change this from HELM_INSTALL_DIR to INSTALL_PATH?